cypress/integration/dashboard/children_spec.js

```diff
@@ -59,4 +59,17 @@ describe('Homepage children', () => {
       .should('have.attr', 'href', '/dashboard/children/u1')
       .invoke('text').should('eq', 'See all 3 badges');
   });
+
+  it('should show the view all children link if they have more than 3 children', () => {
```

Comment:
Add that it should not show for <3 children (parent1)

Reply:
on it!

Reply:
doine